io/plugins/eu.esdihumboldt.hale.io.jdbc.test/src/eu/esdihumboldt/hale/io/jdbc/test/AbstractDBTest.java

```diff
@@ -74,8 +86,8 @@ public AbstractDBTest(DBImageParameters imageParams) {
 	/**
 	 * Setup host and database.
 	 * 
-	 * @throws InterruptedException
-	 * @throws DockerException
+	 * @throws InterruptedException InterruptedException
+	 * @throws DockerException DockerException
```

Comment:
Please provide a meaningful documentation for throws and param or leave it empty (also in other places). Just repeating the Exception type or the parameter class name does not provide any benefit.

For throws it should reflect the reason(s) why a call would fail with that exception, e.g. "if the configuration of the docker client failed" or "if the docker image could not be started".